src/os/os_unix_test.go

```diff
@@ -0,0 +1,81 @@
+// +build darwin linux,!baremetal freebsd,!baremetal
+
+package os_test
+
+import (
+	"fmt"
+	. "os"
+	"testing"
+	"time"
+)
+
+func randomName() string {
+	// fastrand() does not seem available here, so fake it
+	ns := time.Now().Nanosecond()
+	pid := Getpid()
+	return fmt.Sprintf("%d", uint(ns^pid))
```

Comment:
Or `internal/itoa` ?

As for a small rng, we might want to use `Xorshift32` or `XorshiftMult64` from https://github.com/dgryski/trifles/blob/master/inthash/inthash.go

Reply:
internal is not allowed in tests, but strconv makes sense.

I'm not sure it's worth bothering with a small rng in this pull request.
A better random filename generator should land along with TempFile et al, though, and could be used instead of this one.